_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -49,79 +59,40 @@ Vagrant.configure(2) do |config|
 
 end
 
+
 ```
 
 
-Next step to create Vagrantfile and boot up boxes:
+Copy files from [Box](https://cisco.box.com/s/wykoquj1z76gbedq2mj0pgt0sfhwjvhv), change dir to this folder and you are ready to boot up boxes:
 
+```
 mkorshun@MKORSHUN-2JPYH MINGW64 ~/Documents/workCisco/tutorial
-$ vagrant up
+$ ls
+ubuntu.sh*  Vagrantfile  xr-config
 
+mkorshun@MKORSHUN-2JPYH MINGW64 ~/Documents/workCisco/tutorial
+$ vagrant up
+```
 
 
 ### Ubuntu box pre-configuration
 
 To access Ubuntu box just issue the command (no password required):
 ```
-vagrant ssh
-```
-
-
-Install Ansible and all required packages for it. 
-
-```
-sudo apt-get install python-setuptools python-dev build-essential git libssl-dev libffi-dev
-sudo easy_install pip sshpass
-wget https://bootstrap.pypa.io/ez_setup.py -O - | sudo python
-
-git clone http://gitlab.cisco.com/aermongk/iosxr-ansible.git
-git clone git://github.com/ansible/ansible.git --recursive
-cd ansible/ && sudo python setup.py install
-```
-
-Create key to access XR in future:   
-
-```
-vagrant@vagrant-ubuntu-trusty-64:~$ ssh-keygen -t rsa
-Generating public/private rsa key pair.
-Enter file in which to save the key (/home/vagrant/.ssh/id_rsa):
-Enter passphrase (empty for no passphrase):
-Enter same passphrase again:
-.our identification has been saved in
-.pub.public key has been saved in
-The key fingerprint is:
-4e:a5:24:de:e1:51:7c:85:78:a1:65:b1:08:17:e3:be vagrant@vagrant-ubuntu-trusty-64
-The key's randomart image is:
-+--[ RSA 2048]----+
-|        ..=o==.  |
-|         =o*+.   |
-|      . + =o.    |
-|     . = *       |
-|      . S .      |
-|       o   .     |
-|        . E      |
-|                 |
-|                 |
-+-----------------+
-vagrant@vagrant-ubuntu-trusty-64:~$ 
-vagrant@vagrant-ubuntu-trusty-64:~$ cat ~/.ssh/id_rsa.pub
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCfbpi4N3Lcl5i9Y8gd/g4x05IvnIfoPJkhdmGBW2HMFwqWgJQkJF1BM8SuukWeG8+Su4g0Un5tU4/nvbAcqBDR7wFEmB7z7k8VQrXZUxeB4Lc1jEwdDLbxxGOUitLQO+IXlHFJVpkp9Ps6tT82xopaSOQFXKDq0vYXdeEMD/k3NG++0u5pOsJu+kXLIULy1Ix6qvcFDRKbqfT14fU/K7vBYz8gP8Cl+sql9ySm7aOSb0liCBx46/SueBX6uadnMgecVBc1GyvmEX5PwBppUr3Cpby2yOf69iX4NnNcWTTrPY7o7LWuXPNiAgbSAS3R7jVBPyTltB9zCbGq8We5UuVX vagrant@vagrant-ubuntu-trusty-64
-vagrant@vagrant-ubuntu-trusty-64:~$ 
+vagrant ssh ubuntu
 ```
+Ubuntu is already configured via file "ubuntu.sh". Ansible and requred packages are installed.
```

Comment:
Always give small snippets of the shell script to indicate what you're trying to do in ubuntu.sh. Otherwise it becomes cumbersome for readers to go back and forth between the script and steps you describe here, just to understand what is happening.